### PR TITLE
Fix assorted typos

### PR DESCRIPTION
**Pull request by SethTisue**
that I've spotted over the past couple years

### CODE REVIEW

File: compiler/src/dotty/tools/dotc/core/TypeComparer.scala [hunk 1]
```diff
@@ -2838,7 +2838,7 @@ class TypeComparer(@constructorOnly initctx: Context) extends ConstraintHandling
           else
             false
       case (AppliedType(tycon1, args1), AppliedType(tycon2, args2)) if isSame(tycon1, tycon2) =>
-        // It is possible to conclude that two types applies are disjoint by
+        // It is possible to conclude that two types applied are disjoint by
         // looking at covariant type parameters if the said type parameters
         // are disjoin and correspond to fields.
```
Comment (som-snytt) [inline compiler/src/dotty/tools/dotc/core/TypeComparer.scala:2843]: disjoint

File: compiler/src/dotty/tools/dotc/core/TypeComparer.scala [hunk 2]
```diff
@@ -2838,7 +2838,7 @@ class TypeComparer(@constructorOnly initctx: Context) extends ConstraintHandling
           else
             false
       case (AppliedType(tycon1, args1), AppliedType(tycon2, args2)) if isSame(tycon1, tycon2) =>
-        // It is possible to conclude that two types applies are disjoint by
+        // It is possible to conclude that two types applied are disjoint by
```
Comment (som-snytt) [inline compiler/src/dotty/tools/dotc/core/TypeComparer.scala:2841]: it still has the French syntax. "applied types" as in `AppliedType`.

File: compiler/src/dotty/tools/dotc/inlines/Inliner.scala [hunk 3]
```diff
@@ -1047,13 +1047,13 @@ class Inliner(val call: tpd.Tree)(using Context):
     val evaluatedSplice = inContext(quoted.MacroExpansion.context(inlinedFrom)) {
       Splicer.splice(body, splicePos, inlinedFrom.srcPos, MacroClassLoader.fromContext)
     }
-    val inlinedNormailizer = new TreeMap {
+    val inlinedNormalizer = new TreeMap {
```
Comment (som-snytt) [inline compiler/src/dotty/tools/dotc/inlines/Inliner.scala:1050]: Norman Mailer.

File: compiler/src/dotty/tools/dotc/transform/patmat/Space.scala [hunk 4]
```diff
@@ -434,7 +434,7 @@ object SpaceEngine {
    *
    *  We cannot use type erasure here, as it would lose the constraints
    *  involving GADTs. For example, in the following code, type
-   *  erasure would loose the constraint that `x` and `y` must be
+   *  erasure would lose the constraint that `x` and `y` must be
```
Comment (som-snytt) [inline compiler/src/dotty/tools/dotc/transform/patmat/Space.scala:437]: Interestingly, to lose a constraint is to loose the constraint.

**Comment by som-snytt** [comment]
What a nice collection!

File: compiler/src/dotty/tools/dotc/core/TypeComparer.scala (hunk 1 above)
Comment (SethTisue) [reply compiler/src/dotty/tools/dotc/core/TypeComparer.scala:2843]: eh?

File: compiler/src/dotty/tools/dotc/core/TypeComparer.scala (hunk 2 above)
Comment (SethTisue) [reply compiler/src/dotty/tools/dotc/core/TypeComparer.scala:2841]: sorry, I don't follow.

File: compiler/src/dotty/tools/dotc/core/TypeComparer.scala (hunk 1 above)
Comment (som-snytt) [reply compiler/src/dotty/tools/dotc/core/TypeComparer.scala:2843]: `// are disjoin and correspond to fields.`

File: compiler/src/dotty/tools/dotc/core/TypeComparer.scala (hunk 2 above)
Comment (som-snytt) [reply compiler/src/dotty/tools/dotc/core/TypeComparer.scala:2841]: `two applied types are disjoint`

File: compiler/src/dotty/tools/dotc/core/TypeComparer.scala (hunk 1 above)
Comment (SethTisue) [reply compiler/src/dotty/tools/dotc/core/TypeComparer.scala:2843]: oh. true, but this PR doesn't fix every typo 🤷

File: compiler/src/dotty/tools/dotc/core/TypeComparer.scala (hunk 2 above)
Comment (SethTisue) [reply compiler/src/dotty/tools/dotc/core/TypeComparer.scala:2841]: meh

File: compiler/src/dotty/tools/dotc/transform/patmat/Space.scala (hunk 4 above)
Comment (SethTisue) [reply compiler/src/dotty/tools/dotc/transform/patmat/Space.scala:437]: I did spend a few moments pondering if "loosen" might have been meant.

**Review by nicolasstucki — APPROVED** [review]

**Pull request merged**